CHANGELOG.md

```diff
@@ -2,14 +2,16 @@ Next
 ----
 
 * Your contribution here.
+* Adds Grape::Roar::Extensions::RelationalModels
```

Comment:
Yes, sorry! Wanted to get code review out of the way first since it's a big PR and didn't know how many things would be changing 😸